app/Core/Auth.php

```diff
@@ -57,7 +56,7 @@ public function myAuth(array $postData)
         if (isset($postData['user_login']) &&  isset($postData['user_pass'])) {
 
             $connection = new DatabaseConnexion();
-            $usersRepository = new UsersRepository();
+            $usersRepository = new UserRepository;
```

Comment:
```suggestion
            $usersRepository = new UserRepository();
```
Ca me parait suspect come utilisation, j'ai jaais vue

Reply:
Sorry